### PR TITLE
Handle exception that's thrown when quotes.yml doesn't exist.

### PR DESCRIPTION
**Pull request by herbmillerjr**
Handle exception that's thrown when quotes.yml doesn't exist. Create the file with short description.

My first pull request to any project ever. Here goes......

### CODE REVIEW

File: lib/models/quotelist.rb [hunk 1]
```diff
@@ -2,8 +2,12 @@ class QuoteList
   def initialize(config)
     unless config[:quotes_file].nil?
       @quotes_path = File.join File.dirname(__FILE__), "../../#{config[:quotes_file]}"
-      @quotes = YAML.load_file @quotes_path
-      @can_save = true
+	  @can_save = true
```
Comment (eriknelson) [inline lib/models/quotelist.rb]: Can you update to use 2 space indents?

**Comment by eriknelson** [comment]
Makes sense to me pending that indent update 👍thanks!

File: lib/models/quotelist.rb (hunk 1 above)
Comment (herbmillerjr) [inline lib/models/quotelist.rb]: Yup, switched vim settings half way through. Guess I missed a line backtracking.

**Comment by rikai** [comment]
We hang out in #jupiterdev on geekshed if you wanna talk shop at all. :)

**Pull request merged**